resolwe_bio/tools/bamtobigwig.sh

```diff
@@ -5,15 +5,36 @@ source /etc/profile.d/resolwe-base.sh
 bam_file="$1"
 number_of_cores="$2"
 bin_size="$3"
+time="$4"
```

Comment:
Use a more descriptive variable, e.g. `time_limit`. This will make it obviuous to developer that this is a time limit for job execution.